app/org/sagebionetworks/bridge/dynamodb/DynamoSubpopulation.java

```diff
@@ -49,6 +51,16 @@ public DynamoSubpopulation() {
     }
     
     @JsonIgnore
+    @DynamoDBIgnore
+    public String getKey() {
+        return "subpopulation:"+guid;
+    }
+    
+    @Override
+    public void setKey(String key) {
+    }
```

Comment:
We use this (the fact that these fields are set on the subpopulation and are columns in the table). We have to migrate. After deployment, saving each subpopulation should save a criteria object, then we can remove the interface, the fields, etc. and redeploy.